### ISSUE TITLE
[Bug]  Export/Import of expressions does not keep setting "Keep all Cookies"

### ISSUE THREAD

**Issue by farblos** [mrdokenny/Cookie-AutoDelete#1376]
### Acknowledgements

- [X] I acknowledge that I have read the above items

### Describe the bug

Import/Export of expressions does not keep setting "Keep all Cookies": When that has been deselected while exporting for some site, it is selected again after importing the JSON.

Observed with FF "91.8.0esr (64-bit)" , CAD 3.6.0.

### To Reproduce

1. create a new FF profile (version "91.8.0esr (64-bit)" in my case)

2. install CAD into it (version 3.6.0 in my case)

3. go to some site setting cookies, for example github.com

4. create a whitelist entry on "*.github.com"

5. unselect "Keep All Cookies"

6. select one of the cookies below "Keep All Cookies", for example "tz"

7. goto CAD Settings, "List of Expressions"

8. select "Export Expressions..." and save the file locally

9. select "Remove All Expressions", OK the dialog

10. select "Import Expression..." and load the file just saved

### Expected Behavior

"Keep All Cookies" is still unselected on the entry for "*.github.com".

### Screenshots

Observed Behavior: "Keep All Cookies" is selected again on the entry for "*.github.com".

### System Info - Operating System (OS)

Debian 11.3

### System Info - Browser Info

91.8.0esr (64-bit)

### System Info - CookieAutoDelete Version

3.6.0

**Comment by farblos** [comment]
The exported JSON from the example above after step 8. looks like this:

``` json
{
  "default": [
    {
      "expression": "*.github.com",
      "listType": "WHITE",
      "storeId": "default",
      "cleanAllCookies": false,
      "cleanSiteData": [],
      "cookieNames": [
        "tz"
      ],
      "id": "4bd6rvcDm"
    }
  ]
}
```

**Comment by kennethtran93** [comment]
This is included in 3.7.0 and has been released on Firefox.

**Comment by farblos** [comment]
Thanks a lot for fixing this so quickly!
